CRM/Utils/PDF/Utils.php

```diff
@@ -72,7 +88,7 @@ public static function html2pdf($text, $fileName = 'civicrm.pdf', $output = FALS
 <html>
   <head>
     <meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\"/>
-    <style>@page { margin: {$t}{$metric} {$r}{$metric} {$b}{$metric} {$l}{$metric}; }</style>
+    <style>@page { margin: {$t}{$metric} {$r}{$metric} {$b}{$metric} {$l}{$metric}; $css_page_size}</style>
```

Comment:
Sorry not the ending space, the one in front of $css_page_size.

Reply:
Thanks for your patience. I didn't read your previous comment closely enough last night.